public/apps/configuration/configuration.html

```diff
@@ -113,7 +113,7 @@ <h2 class="kuiSubTitle" style="margin-bottom:10px;">Authentication Backends</h2>
                             <h2 class="kuiSubTitle" style="margin-bottom:10px;">System</h2>
 
                             <div class="FeaturePanelList">
-                                <a id="opendistro_security.link.securityconfig" ng-if="endpointAndMethodEnabled('SECURITYCONFIG', 'GET')" class="FeaturePanel ng-scope"
+                                <a id="opendistro_security.link.securityconfig" ng-if="endpointAndMethodEnabled('CONFIG', 'GET')" class="FeaturePanel ng-scope"
```

Comment:
Curious why this value was changed.

Reply:
This is a bug fix. The API is named CONFIG on the backend. It was querying incorrectly and showing "Authentication & Authorizations" page incorrectly to the users.